modules/distribution/src/repository/resources/dbscripts/apimgt/oracle_rac.sql

```diff
@@ -114,13 +116,13 @@ CREATE TABLE IDN_OAUTH2_SCOPE (
             NAME VARCHAR2 (255) NULL,
             DESCRIPTION VARCHAR2 (512) NULL,
             TENANT_ID INTEGER DEFAULT 0,
-        ROLES VARCHAR2 (500) NULL,
+	    ROLES VARCHAR2 (500) NULL,
```

Comment:
Directly copied content from https://github.com/wso2/carbon-apimgt/tree/v5.0.4/features/apimgt/org.wso2.carbon.apimgt.core.feature/src/main/resources/sql as apim version upgraded to v5.04